_apps/all-contributors.m

```diff
@@ -0,0 +1,22 @@
+---
+title: All Contributors
+description: Easily recognize all contributions per the all-contributors spec
+slug: allcontributors
+screenshots:
+- https://raw.githubusercontent.com/all-contributors/all-contributors-bot/master/docs/usage.png
+authors:
+- jakebolam
+repository: all-contributors/all-contributors-bot
+stars: 39
+updated: 2019-01-21 18:48:00 UTC
+host: https://gkioebvccg.execute-api.us-east-1.amazonaws.com/prod
```

Comment:
@JasonEtco for the sync to work. Do the stats need to be at `https://gkioebvccg.execute-api.us-east-1.amazonaws.com/prod/stats/probot` or  is`https://gkioebvccg.execute-api.us-east-1.amazonaws.com/prod/stats` fine?

Reply:
If you enter `host: https://gkioebvccg.execute-api.us-east-1.amazonaws.com/prod` we expect the stats to be at https://gkioebvccg.execute-api.us-east-1.amazonaws.com/prod/probot/stats

Reply:
Thanks, @hiimbex. I've updated the endpoint path to be at the expected location:
https://github.com/all-contributors/all-contributors-bot/commit/a55498562a169d77d734c1c155b1e09339d3a95c